### PR TITLE
refactor get method to allow geting nested properties

### PR DESCRIPTION
**Pull request by pociej**

### CODE REVIEW

**Comment by mitar** [comment]
Hm, this is not reactive?

**Comment by pociej** [comment]
not sure if understand you @mitar . Is previous implentation reactive ? How you want to make it reactive if parent property can be everything , in most cases it isnt reactive datasource .

**Comment by mitar** [comment]
Hm, yea, but why is then this needed? Then there is no much difference between `template.get('a.b')` and `template.get('a').b`? Except that you make field names not be able to contain dots?

File: template-extension.js [hunk 1]
```diff
@@ -235,15 +235,29 @@ Template.prototype.copyAs = function (newTemplateName) {
...
     template = template.parent(1, true);
   }
 };
 
+
+function hasProperty(obj, prop) {
+  var parts = prop.split(".");
+  var cur = obj;
+  for (var i=0; i<parts.length; i++) {
+    if (!cur[parts[i]])
```
Comment (mitar) [inline template-extension.js]: What if property exists, but it is not true?

**Comment by mitar** [comment]
Oh, I see, because it goes up to the parent if it does not exist. Hmm.

**Comment by pociej** [comment]
yep its the reason

File: template-extension.js (hunk 1 above)
Comment (aldeed) [inline template-extension.js]: Agree, should be something like `if (!cur.hasOwnProperty(parts[i]))`

And add an additional test where the value is `false` to make sure it returns `false`.

**Comment by aldeed** [comment]
@pociej See one comment, but then I think we could merge this. @mitar do you agree?

**Comment by mitar** [comment]
I think it is OK. But we should document this.

**Comment by pociej** [comment]
hi @mitar , @aldeed  thx for comment will fix today evening .

**Pull request closed without merging**